model_analyzer/model_manager.py

```diff
@@ -263,21 +263,33 @@ def _create_model_variant(self, original_name, variant_config):
                 pass
 
     def _load_model_variant(self, variant_config):
+        """
+        Decides on whether or not to load a model variant in the client
```

Comment:
This implies that it only decides IF it should be loaded. It does however still do loading in some (most) cases.  Maybe `conditionally loads a model variant in the client`?

Reply:
Nice.  Done.